### PR TITLE
Maven plugin upgrade

### PR DESCRIPTION
**Pull request by josemduarte**
Upgrading maven plugin versions. Fixed a few tests that were broken when running on maven.

### CODE REVIEW

File: biojava-integrationtest/src/test/java/org/biojava/nbio/structure/test/symmetry/TestQuatSymmetryDetectorExamples.java [hunk 1]
```diff
@@ -223,20 +223,22 @@ public void testLocal() throws IOException, StructureException {
...
+//				localSymmetries.put("G5","H");
+//			testLocalSymmetries.add(localSymmetries);
 
 		QuatSymmetryParameters symmParams = new QuatSymmetryParameters();
 		SubunitClustererParameters clusterParams = new SubunitClustererParameters(true);
 		clusterParams.setClustererMethod(SubunitClustererMethod.SEQUENCE);
 		clusterParams.setSequenceIdentityThreshold(0.75);
 
 		for(int iTest = 0; iTest<testIds.size();iTest++) {
+			System.out.println(testIds.get(iTest));
```
Comment (lafita) [inline biojava-integrationtest/src/test/java/org/biojava/nbio/structure/test/symmetry/TestQuatSymmetryDetectorExamples.java]: Should this be using logger?

File: biojava-integrationtest/src/test/java/org/biojava/nbio/structure/test/symmetry/TestQuatSymmetryDetectorExamples.java [hunk 2]
```diff
@@ -249,6 +251,7 @@ public void testLocal() throws IOException, StructureException {
 			Map<String,String> refLocal = testLocalSymmetries.get(iTest);
 
 			for (QuatSymmetryResults local:foundLocal) {
+				System.out.println(local.getStoichiometry().toString());
```
Comment (lafita) [inline biojava-integrationtest/src/test/java/org/biojava/nbio/structure/test/symmetry/TestQuatSymmetryDetectorExamples.java]: Logger.info?

File: biojava-structure/src/test/resources/log4j2.xml [hunk 3]
```diff
@@ -9,5 +9,13 @@
 		<root level="warn">
 			<appender-ref ref="Console"/>
 		</root>
+		<!--
```
Comment (lafita) [inline biojava-structure/src/test/resources/log4j2.xml]: Do you have to include this comment?

**Comment by josemduarte** [comment]
Thanks @lafita, I've pushed a commit

**Pull request merged**